cli/src/commands.rs

```diff
@@ -263,13 +263,21 @@ pub(crate) async fn handle_command(
                 })
                 .await?;
 
-            wait_confirmation!(
-                format!(
-                    "Fees: {} sat. Are the fees acceptable? (y/N) ",
-                    prepare_response.fees_sat
-                ),
-                "Payment receive halted"
-            );
+            let fees = prepare_response.fees_sat;
+            let confirmation_msg = match payer_amount_sat {
+                Some(_) => format!("Fees: {fees} sat. Are the fees acceptable? (y/N)"),
+                None => {
+                    let min = prepare_response.zero_amount_min_payer_amount_sat.unwrap();
+                    let max = prepare_response.zero_amount_max_payer_amount_sat.unwrap();
+                    let service_feerate = prepare_response.zero_amount_service_feerate.unwrap();
+                    format!(
+                        "Fees: {fees} sat + {service_feerate}% of the sent amount. \
```

Comment:
I think the service feerate is going to be a `0.1` type value

Reply:
Yes, the service feerate is already the percentage value (i.e. `0.1 %`).